{"rendered": {"message": {"raw": "Issue #34", "markup": "markdown", "html": "<p>Issue <a href=\"#!/levitsky/pyteomics/issues/34/parser-fails-on-numbers-in-modx-sequence\" rel=\"nofollow\" title=\"parser fails on numbers in modX sequence\" class=\"ap-connect-link\"><s>#34</s></a></p>", "type": "rendered"}}, "hash": "6af434bde9f67795745f237afe6af7b8daf3d6ce", "repository": {"links": {"self": {"href": "data/repositories/levitsky/pyteomics.json"}, "html": {"href": "#!/levitsky/pyteomics"}, "avatar": {"href": "data/bytebucket.org/ravatar/{3f6302de-10da-4d3c-b258-0bb6a2368972}ts=74456"}}, "type": "repository", "name": "pyteomics", "full_name": "levitsky/pyteomics", "uuid": "{3f6302de-10da-4d3c-b258-0bb6a2368972}"}, "links": {"self": {"href": "data/repositories/levitsky/pyteomics/commit/6af434bde9f67795745f237afe6af7b8daf3d6ce.json"}, "comments": {"href": "data/repositories/levitsky/pyteomics/commit/6af434bde9f67795745f237afe6af7b8daf3d6ce/comments_page=1.json"}, "patch": {"href": "https://api.bitbucket.org/2.0/repositories/levitsky/pyteomics/patch/6af434bde9f67795745f237afe6af7b8daf3d6ce"}, "html": {"href": "#!/levitsky/pyteomics/commits/6af434bde9f67795745f237afe6af7b8daf3d6ce"}, "diff": {"href": "https://api.bitbucket.org/2.0/repositories/levitsky/pyteomics/diff/6af434bde9f67795745f237afe6af7b8daf3d6ce"}, "approve": {"href": "https://api.bitbucket.org/2.0/repositories/levitsky/pyteomics/commit/6af434bde9f67795745f237afe6af7b8daf3d6ce/approve"}, "statuses": {"href": "data/repositories/levitsky/pyteomics/commit/6af434bde9f67795745f237afe6af7b8daf3d6ce/statuses_page=1.json"}}, "author": {"raw": "Lev Levitsky <lev.levitsky@phystech.edu>", "type": "author", "user": {"display_name": "Lev Levitsky", "uuid": "{eb44325f-4ee0-4e0b-a27c-f2ea23122a56}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Beb44325f-4ee0-4e0b-a27c-f2ea23122a56%7D"}, "html": {"href": "https://bitbucket.org/%7Beb44325f-4ee0-4e0b-a27c-f2ea23122a56%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/a2593c44c42429c503d2e5e9e307e241d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsLL-6.png"}}, "nickname": "levitsky", "type": "user", "account_id": "557058:986c547b-c50a-40b3-948a-29b4a93b7b30"}}, "summary": {"raw": "Issue #34", "markup": "markdown", "html": "<p>Issue <a href=\"#!/levitsky/pyteomics/issues/34/parser-fails-on-numbers-in-modx-sequence\" rel=\"nofollow\" title=\"parser fails on numbers in modX sequence\" class=\"ap-connect-link\"><s>#34</s></a></p>", "type": "rendered"}, "participants": [], "parents": [{"hash": "408f4e84d32598d8d163257eb7514fe392ee75f9", "type": "commit", "links": {"self": {"href": "data/repositories/levitsky/pyteomics/commit/408f4e84d32598d8d163257eb7514fe392ee75f9.json"}, "html": {"href": "#!/levitsky/pyteomics/commits/408f4e84d32598d8d163257eb7514fe392ee75f9"}}}], "date": "2018-11-13T14:48:47+00:00", "message": "Issue #34", "type": "commit", "git_hash": "dffff6cfc27d6413909805b4e4649fe13cc235fc", "tags": null, "branches": "4.0"}